.github/CONTRIBUTING.md

```diff
@@ -1,67 +1,30 @@
 # Contributing
 
-Thank you for considering to help Wagtail.
-
-We welcome all support, whether on bug reports, code, design, reviews, tests,
-documentation, translations or just feature requests.
+Thank you for your interest in improving Wagtail! There are many ways to contribute - see the [Contributing](https://docs.wagtail.org/en/latest/contributing/index.html) documentation for some suggestions.
 
 ## Working on an issue
 
-👉 If an issue isn’t being worked on by anyone, go for it! **No need to ask "please assign me this issue".** Add a comment to claim the issue once you’re ready to start.
-
-Please review the [contributing guidelines](https://docs.wagtail.org/en/latest/contributing/index.html).
-You might like to start by checking issues with the [good first issue](https://github.com/wagtail/wagtail/labels/good%20first%20issue) label.
-
-## Reporting bugs
-
-To report bugs, use our [issue tracker](https://github.com/wagtail/wagtail/issues).
-
-## Feature requests
+👉 If an issue isn’t being worked on by anyone, go for it! **No need to ask "please assign me this issue".** Feel free to leave a comment outlining your plans so that other people know you're working on it.
 
-Use our [issue tracker](https://github.com/wagtail/wagtail/issues) for feature requests, or go the [Wagtail Slack](https://github.com/wagtail/wagtail/wiki/Slack) or [Discussions](https://github.com/wagtail/wagtail/discussions) if you want to discuss an idea before requesting it.
+If you're new to contributing to open source, we recommend you read the [Your first contribution guide](https://docs.wagtail.org/en/latest/contributing/first_contribution_guide.html). You might like to start by checking issues with the [good first issue](https://github.com/wagtail/wagtail/labels/good%20first%20issue) label.
 
-## Support
+Please review the [contributing guidelines](https://docs.wagtail.org/en/latest/contributing/index.html) for details of [setting up a development environment](https://docs.wagtail.org/en/latest/contributing/developing.html) and other coding guidelines. In summary:
 
-Please don't use the issue tracker for support - use [the 'wagtail' tag on Stack Overflow](https://stackoverflow.com/questions/tagged/wagtail) (preferred) or the [#support channel](https://github.com/wagtail/wagtail/wiki/Slack#support) on the [Wagtail Slack](https://github.com/wagtail/wagtail/wiki/Slack).
+- **Purely AI-generated contributions are not welcome.** If you have made use of AI assistance, please disclose this in your pull request description, and detail the steps you have taken to verify that the code is correct. See the guidelines for [use of generative AI](https://docs.wagtail.org/en/latest/contributing/general_guidelines.html#use-of-generative-ai).
```

Comment:
I'd like contributors to understand their contributions and take ownership.
I'm not sure how to word this, or how contributors can prove it.